src/skin/legacy/legacyskinparser.cpp

```diff
@@ -1049,7 +1067,7 @@ QWidget* LegacySkinParser::parseText(const QDomElement& node) {
     QString group = lookupNodeGroup(node);
     BaseTrackPlayer* pPlayer = m_pPlayerManager->getPlayer(group);
     if (!pPlayer) {
-        SKIN_WARNING(node, *m_pContext) << "No player found for group:" << group;
+        SKIN_WARNING(node, *m_pContext, QStringLiteral("No player found for group:").arg(group));
```

Comment:
Here is the %1 missing. 
I am afraid the arg() version is more expensive than the original.  
I have an idea to keep the stream type working. See my other comment.

Reply:
While I understand the argument to aim for best possible performance and appreciate the effort you put into the skinWarning class example below, I do wonder if this fully applies here. It's a warning after all indicating the GUI won't be functional as expected, and if a significant amount of warnings is printed this small performance regression is the least we should be worried about IMHO.

Reply:
This does not apply here, sure. The mandatory issue is only the missing %1